apps/condo/domains/notification/adapters/apple/session.js

```diff
@@ -0,0 +1,113 @@
+const http2 = require('http2')
+
+const { isEmpty, get } = require('lodash')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+
+const { getCurrSec } = require('@condo/domains/common/utils/date')
+
+const SESSION_LIFE_TIME = 60 * 60 * 24
+const SESSION_PING_INTERVAL = 60
+// const APPLE_API_ENDPOINT = 'https://api.sandbox.push.apple.com'
```

Comment:
Debug line?
```suggestion
```

Reply:
will be removed later, after production support added on mobile apps.